test/basic-querying.test.js

```diff
@@ -168,7 +173,8 @@ describe('basic-querying', function() {
       });
     });
 
-    it('should query collection with skip & limit', function(done) {
+    bdd.itIf(connectorCapabilities.supportPagination !== false, 'should query collection with skip & ' +
```

Comment:
This could either be `connectorCapabilities.supportPagination` or `!!connectorCapabilities.supportPagination` if you need to cast an undefined to a boolean.